src/test/java/com/google/javascript/clutz/ctor_func.d.ts

```diff
@@ -1,5 +1,7 @@
 declare namespace ಠ_ಠ.clutz.ctor_func {
-  class Ctor < T > {
+  class Ctor < T > extends Ctor_Instance < T > {
```

Comment:
Nit: something we could consider in a later commit is checking if a class has any static fields at all, and if not, merge the instance and static classes into just `Ctor` here. I presume that at some point, users will read these `.d.ts` files to try and make sense of their code, so less noise might be nicer.